codalab/lib/worksheet_util.py

```diff
@@ -749,11 +761,22 @@ def raise_usage_error():
         else:
             raise UsageError('Unknown display mode: %s' % mode)
         bundle_infos[:] = []  # Clear
+        return (start_index, focus_index + 1, -1)
 
     def get_command(value_obj):  # For directives only
         return value_obj[0] if len(value_obj) > 0 else None
 
-    last_was_empty_line = False
+    def update_interpreted_raw_map(start_index, end_index, focus_index, sub_focus_index=-1):
+        interpreted_raw_map[str(focus_index) + ',' + str(sub_focus_index)] = end_index
+        raw_interpreted_list.append(((focus_index, sub_focus_index), end_index))
+
+    def update_indexes(end_index, focus_index):
```

Comment:
nit: This function has no side-effects, so `updated_indices` might make more sense.